StripePaymentSheet/StripePaymentSheet/Source/Categories/STPPaymentMethod+PaymentSheet.swift

```diff
@@ -47,12 +47,47 @@ extension STPPaymentMethod {
             return nil
         }
     }
-    
+
     func paymentOptionLabel(confirmParams: IntentConfirmParams?) -> String {
         if let instantDebitsLinkedBank = confirmParams?.instantDebitsLinkedBank {
             return "••••\(instantDebitsLinkedBank.last4 ?? "")"
         } else {
             return paymentSheetLabel
         }
     }
+
+    func hasUpdatedCardParams(_ updatedParams: STPPaymentMethodCardParams?) -> Bool {
+        guard let currCard = self.card,
+              let updatedParams = updatedParams else {
+            return false
+        }
+        let updatedExpMM = NSNumber(value: currCard.expMonth) != updatedParams.expMonth
+        let updatedExpYY = currCard.twoDigitYear != updatedParams.expYear
+
+        return updatedExpMM || updatedExpYY
+    }
+
+    func hasUpdatedAutomaticBillingDetailsParams(_ updatedParams: STPPaymentMethodBillingDetails?) -> Bool {
+        guard let updatedParams = updatedParams else {
+            return false
+        }
+        let updatedCountry = self.billingDetails?.address?.country != updatedParams.address?.country
```

Comment:
Is it possible for the updatedParams to "unset" things. For example maybe `self.billingDetails?.address?.country` is "US", but `updatedParams.address?.country` is nil? Is this possible/valid? I wonder if we should unwrap `updatedParams` and it's properties before comparing if we don't intend to support "unsetting" things.

Reply:
Great callout -- for now, there was no plan to 'Unset" any billing address items.  I'll start a thread in slack to discuss further.